spec/services/waste_carriers_engine/order_copy_cards_completion_service_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteCarriersEngine
+  RSpec.describe OrderCopyCardsCompletionService do
+    describe ".run" do
+      let(:finance_details) { double(:finance_details) }
+      let(:transient_finance_details) { double(:transient_finance_details) }
+      let(:registration) { double(:registration, finance_details: finance_details) }
+      let(:transient_registration) do
+        double(
+          :transient_registration,
+          registration: registration,
+          finance_details: transient_finance_details
+        )
+      end
+
+      it "completes an order" do
+        orders = double(:orders)
+        payments = double(:payments)
+        transient_order = double(:transient_order)
+        transient_payment = double(:transient_payment)
+
+        # Creates a past registration
```

Comment:
The method is definitely being called, but as I said In my previous comment, I don't believe it is actually creating a past registration.

So I think this test is not right, but it doesn't matter as I don't think we should be doing this anyway!

Reply:
Good! I have copied it over from the renewal completion service, I have no strong feelings on keeping it so happy to remove if Iris say so.

Reply:
Let's just go ahead and remove it. It was more my reading of `build_past_registration()` I was after a second opinion on (and that in this scenario it would always return null and not create anything).

However, I'm happy to state we should not be creating a past registration record when just taking a copy card order. So go ahead and remove it from the service and the spec and then I'm good to approve.